modules/hub-legacy/main.tf

```diff
@@ -72,7 +72,7 @@ resource "google_service_account_key" "gke_hub_key" {
 
 module "gke_hub_registration" {
   source  = "terraform-google-modules/gcloud/google"
-  version = "~> 3.1"
+  version = "~> 2.1.0"
```

Comment:
Are we downgrading the gcloud module version?

Reply:
No, reverted.